modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -3312,21 +3312,32 @@ export class Wallet implements IWallet {
       throw new Error('txRequestId missing from signed transaction');
     }
 
-    // TODO: BG-51122 Remove conditional when moved to txRequestFull for everything
-    if (this._wallet.type === 'custodial') {
-      await this.bitgo
+    if (onlySupportsTxRequestFull || apiVersion === 'full') {
```

Comment:
what if apiVersion is not passed for eddsa custodial wallet?

Reply:
Then “onlySupportsTxRequestFull” should be true.